routing/pathfind.go

```diff
@@ -587,6 +586,7 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 		// from the heap.
 		partialPath := heap.Pop(&nodeHeap).(*nodeWithDist)
 		pivot := partialPath.node
+		pivotWithDist := distance[pivot]
```

Comment:
how is this different from `partialPath`?

Reply:
it's not, removed!